src/libxively/io/fs/xi_fs_api.h

```diff
@@ -10,47 +10,8 @@
 #include <stdlib.h>
```

Comment:
ok, done, but we should do it for every file, or we wait 5 months and 2017 will be gone.